client/src/pages/Upcoming/index.tsx

```diff
@@ -0,0 +1,57 @@
+import UpcomingBox from "components/UpcomingBox";
+
+import AppToolbar from '@material-ui/core/Toolbar';
+import GithubIcon from '@material-ui/icons/GitHub';
+import { Grid, Box, Card, CardMedia, CardContent, Typography, Slide, AppBar, Button } from '@mui/material';
+import JobCardList from "components/JobCardList";
+import './style.css';
+
+const Upcoming = () => {
+    return (
+        <>
+            <Slide direction='down' in={true} timeout={800}>
+                <AppBar position='sticky' className='AppBar' color='inherit'>
+                    <AppToolbar>
+                    <div className='HeaderContainer'>
```

Comment:
The header does not belong on this page...
The drawer will eventually be here, but we're distinctly not using a nav bar for our site, like we've spoke about

Reply:
+1